.github/workflows/install.yml

```diff
@@ -0,0 +1,116 @@
+name: Installation
+on:
+  push:
+    paths:
+      - "install/**"
+  pull_request:
+    paths:
+      - "install/**"
+
+jobs:
+  check:
+    name: check
+    runs-on: ubuntu-20.04
+    container: koalaman/shellcheck-alpine:stable
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: shell
+        run: shellcheck -e SC1090,SC1091,SC1071 install/install.sh
```

Comment:
There's a comment in `install.sh` to disable `SC2039`. Why are these warning disabled  only here?

Reply:
1090 and 1091 have to do with spellcheck being able to follow sourced or exec'd scripts, which it can't do when things are executed from a url, download, or uncompressed asset. These could be moved into the script, but it's fairly common to disable these for scripts that bootstrap other scripts over the wire.

1071 isn't actually necessary here, but it "ignores" scripts that shellcheck can't lint based on the shebangs (like zsh, for example). We don't have any of those in this use case.

It's a style choice and I'm happy to go either way.

Reply:
In that case a comment in your future PR would be nice.